test/apps/iodemo/io_demo.cc

```diff
@@ -1897,6 +1906,13 @@ static void adjust_opts(options_t *test_opts) {
 
     test_opts->chunk_size = std::min(test_opts->chunk_size,
                                      test_opts->max_data_size);
+
+    // randomize servers to optimize startup
+    IoDemoRandom::srand(std::time(NULL) ^ getpid());
+    std::random_shuffle(test_opts->servers.begin(), test_opts->servers.end(),
+                        IoDemoRandom::urand<size_t>);
+    // restore seed
+    IoDemoRandom::srand(test_opts->random_seed);
```

Comment:
Why not use user's seed for the random shuffle as well?
we want all random decisions could be reproducible..

Reply:
if all clients do shuffle with the same seed then it won't take any effect, they will connect in the same order as before

Reply:
yes, but we don't have to pass the same seed to all clients...

Reply:
do you propose to generate seed on runner side per client? IMHO this is not obvious from user's perspective

Reply:
agree and when using a runner script it can get tricky
but IMO better allow users to control a seed in order to reproduce issues, 
i.e check which seed was generated randomly in each process by looking at the log, and pass it as arg

Reply:
makes sense, so I'll add a printing of this seed and another command line parameter. this is what you are proposing, right?
BTW maybe print resulting servers list as well?

Reply:
i think seed is already printed
server list - yes let's print